ml-core-cardscan/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply from: configs.androidLibrary
+
+dependencies {
+    api project(':ml-core-base')
+    implementation project(":stripecardscan-tflite")
```

Comment:
`ml-core-cardscan` uses the tflite aar specifically built for stripecardscan. Since only cardscan is using this dependency, we don't need to "leak" support library here.